eth-custodian/contracts/EthCustodian.sol

```diff
@@ -59,15 +59,15 @@ contract EthCustodian is ProofKeeper {
         returns (BurnResult memory result)
     {
         Borsh.Data memory borshData = Borsh.from(data);
-        result.amount = borshData.decodeU128();
         bytes20 recipient = borshData.decodeBytes20();
+        result.amount = borshData.decodeU128();
         result.recipient = address(uint160(recipient));
```

Comment:
Why changing the order here? is the counter part of this code already building the proof the other way?

In original token connector the data was [passed as](https://github.com/near/rainbow-token-connector/blob/master/bridge-token-factory/src/lib.rs#L230-L234): `amount`, `token`, `recipient`

Reply:
This is to make it more consistent with both Deposit and Withdraw events (addresses come first). The counter part for eth-connector afaik hasn't been implemented yet, so I think we can design it whatever we think is more consistent and convenient.